pkg/clusteragent/admission/mutate/autoscaling/autoscaling.go

```diff
@@ -52,6 +56,11 @@ func (w *Webhook) Name() string {
 	return w.name
 }
 
+// WebhookType returns the type of the webhook
+func (w *Webhook) WebhookType() common.WebhookType {
+	return w.webhookType
+}
```

Comment:
`autoscaling.Webhook` can only be of one possible type. Having a field consuming memory to store it may not be needed.

```suggestion
// WebhookType returns the type of the webhook
func (w *Webhook) WebhookType() common.WebhookType {
	return common.MutatingWebhook
}
```

Reply:
Fixed 👍